docs/topics/testing/advanced.txt

```diff
@@ -781,6 +781,41 @@ utility methods in the ``django.test.utils`` module.
     configuration that need to be reversed. It's the return value of the
     :meth:`setup_databases` method.
 
+.. function:: isolate_apps(*app_labels, attr_name=None, kwarg_name=None)
+
+    Register the models defined within a wrapped context into their own
+    isolated app registry. This functionality is useful when creating a
+    Django model for a test case. An isolated registry avoids mixing
+    concerns as well as edge cases involving name collision.
+
+    The app labels which the isolated registry should contain must be
+    passed as individual arguments. And the invocation can be either a
+    decorator or a context manager. For example::
```

Comment:
not great to start a sentence with "and". perhaps:

"You can use ``isolate_apps`` as a decorator or a context manager."

Reply:
👍